packages/shared/lib/services/proxy.service.ts

```diff
@@ -146,7 +146,14 @@ class ProxyService {
             endpoint = endpoint.replace(provider.proxy.base_url, '');
         }
 
-        if (headers && headers['Content-Type'] === 'multipart/form-data') {
+        const headersCleaned: Record<string, string> = {};
+        if (headers) {
+            for (const [key, value] of Object.entries(headers)) {
+                headersCleaned[key.toLocaleLowerCase()] = value;
+            }
+        }
```

Comment:
reduce would do the job as well
```
cons headersCleaned = Object.entries(headers || {}).reduce(
    (acc, [key, value]) => ({ ...acc, [key.toLowerCase()]: value }), 
    {}
);
```

Reply:
If I could remove one feature from javascript it would be this one, I hate this function so much I found it unreadable :D 
plus it's slow for no reason https://www.measurethat.net/Benchmarks/Show/32914/2/object-for-of-vs-reduce